pkg/util/workqueue/default_rate_limiters.go

```diff
@@ -38,7 +38,7 @@ type RateLimiter interface {
 // both overall and per-item rate limitting.  The overall is a token bucket and the per-item is exponential
 func DefaultControllerRateLimiter() RateLimiter {
 	return NewMaxOfRateLimiter(
-		DefaultItemBasedRateLimiter(),
+		NewItemExponentialFailureRateLimiter(5*time.Millisecond, 1000*time.Second),
```

Comment:
> It seems that @deads2k has strong opinion that this value should be high.

@smarterclayton does too, since we want to have controllers retry on external conditions that are eventually fixed, but not loop quickly on them.  An image import that fails 20 times for instance.  Maybe it comes back up.  May as well try infrequently, especially since we're talking about removing resync-ing.